security/pass/Portfile

```diff
@@ -39,28 +39,12 @@ use_configure       no
 build {}
 
 destroot.env-append PREFIX=${prefix} \
-                    SYSCONFDIR=${prefix}/etc
+                    SYSCONFDIR=${prefix}/etc \
+                    WITH_ALLCOMP=yes
 
-default_variants    +bash
-
-variant bash description {Include Bash completion support} {
-    depends_run-append port:bash
-    destroot.env-append WITH_BASHCOMP=yes
-
-    notes-append {
-        To use pass bash completion, add the following lines at the end of your .bash_profile:
-            [[ -r "${prefix}/share/bash-completion/completions/pass" ]] && source "${prefix}/share/bash-completion/completions/pass"
-    }
-}
-
-variant fish description {Include fish completion support} {
-    depends_run-append port:fish
-    destroot.env-append WITH_FISHCOMP=yes
-}
-
-variant zsh description {Include Zsh completion support} {
-    depends_run-append port:zsh
-    destroot.env-append WITH_ZSHCOMP=yes
+notes {
+    To use pass bash completion, add the following lines at the end of your .bash_profile:
+        [[ -r "${prefix}/share/bash-completion/completions/pass" ]] && source "${prefix}/share/bash-completion/completions/pass"
```

Comment:
are there also instructions for the other shells? Since you know install all of them, it would be nice to add instructions for `fish` and `zsh` as well.

Reply:
Thanks 🌟, now it is better?

Reply:
thanks - now there are still two lint warnings and then we're good to go ;)
```
Warning: Line 52 has trailing whitespace before newline
Warning: Line 53 has trailing whitespace before newline
```

Reply:
Thanks, done.